src/datepicker/adapters/ngb-date-adapter.ts

```diff
@@ -1,6 +1,6 @@
 import {Injectable} from '@angular/core';
 import {NgbDateStruct} from '../ngb-date-struct';
-import {isNumber} from 'util';
+import {isInteger} from '../../util/util';
```

Comment:
`isNumber` from `util` actually corresponds to `isInteger` from our `../../util/util`...